docs/getting-started-guide-mtr/master.adoc

```diff
@@ -9,14 +9,14 @@ include::topics/templates/document-attributes.adoc[]
 :getting-started-guide:
 
 :_content-type: ASSEMBLY
-= Introduction to the Migration Toolkit for Runtimes
+= Introduction to the {ProductName}
 
 //Inclusive language statement
 include::topics/making-open-source-more-inclusive.adoc[]
 
 == Introduction
 
-// About the {IntroToMTABookName}
+// About the {IntroToMTRBookName}
 include::topics/about-the-intro-to-mta-guide.adoc[leveloffset=+2]
```

Comment:
No need to change commented out lines; not sure this is theoretically correct, but leave it alone due to time.